tests/integration/components/selection-test.js

```diff
@@ -585,7 +585,7 @@ module('Integration | selection', () => {
         let rows = generateRows(1, 1);
         await generateTable(this, { rows });
 
-        this.set('selection', [...rows, { fakeRow: true }]);
+        run(() => this.set('selection', [...rows, { fakeRow: true }]));
```

Comment:
Adding the `run` around the `set` was required when using async observer (ie Ember 3.13+).
However in the collapse-tree tests below I had to use `await settled()` to get those tests to pass — wrapping the changes in a `run` was not sufficient.

Reply:
Wouldn't the end of a runloop still force all the async observers to flush?

https://github.com/emberjs/ember.js/blob/7d86e1242b9655f4995068ec2d189e46076517f5/packages/%40ember/runloop/index.js#L20

Or is the issue that the async observer's run `onEnd` and so if they schedule into a queue the existing runloop is not re-used?

Is that ideal?